src/pug/includes/body.pug

```diff
@@ -1,7 +1,8 @@
 include ./fork.pug
 
 .container
-
+	span#color-switch
```

Comment:
No hardcoded default value is good so if Javascript is not enabled it won't polute the space with useless buttons

Reply:
I didn't get that exactly :/

Reply:
Just said it was good like that :+1: